srcpkgs/mopidy/template

```diff
@@ -1,21 +1,21 @@
 # Template file for 'mopidy'
 pkgname=mopidy
-version=2.3.1
-revision=2
+version=3.0.1
+revision=1
 archs=noarch
 wrksrc="Mopidy-${version}"
-build_style=python2-module
+build_style=python3-module
 pycompile_module=mopidy
-hostmakedepends="python-setuptools python-Sphinx python-sphinx_rtd_theme python-pykka"
+hostmakedepends="python3-setuptools python3-Sphinx python3-sphinx_rtd_theme python-pykka"
```

Comment:
python-pykka -> python3-pykka

Reply:
In fact, python-pykka is a python2 and python3 compatible package. There is no python3-pykka cause the python-pykka one is enough

Reply:
We split python2 and python3, in which the comment about missing the python3-pykka_package() part does, and reason also reason the need to change the dependency from python-pykka to python3-pykka.

Reply:
Understood